.devcontainer/README.md

```diff
@@ -0,0 +1,20 @@
+Codespaces - ISPC developer's environment (CPU only)
+====================================================
+
+This is the minimal environment that enables CPU-only ISPC build (i.e. GPU backend is not enabled).
+
+To build ISPC and run the tests do the following:
+```bash
+# `build` folder already contains result of `cmake .. -DCMAKE_EXPORT_COMPILE_COMMANDS=ON`
+cd build
+# build ISPC and examples
+make -j4
+# run LIT tests
+make check-all
+```
```

Comment:
The contents of the root directory does not contain `build`:
```shell
@e3m3 ➜ /workspaces/ispc (main) $ ls
CMakeLists.txt   alloy.py            check_env.py     common.py    examples      perf.py       stdlib2cpp.py       tests_errors
CONTRIBUTING.md  benchmarks          check_format.sh  contrib      fail_db.txt   run_tests.py  test_static.cpp     third-party-programs.txt
LICENSE.txt      bitcode2cpp.py      check_isa.cpp    docker       ispcrt        scripts       test_static.isph    utils
README.md        builtins            cmake            docs         llvm_patches  src           test_static_l0.cpp
SECURITY.md      check_copyright.sh  common           doxygen.cfg  perf.ini      stdlib.ispc   tests
```

Reply:
Check you build logs, container build probably failed. Before it's merged you need to manually set my repo and my branch as a source.